packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/link-list/link-list.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { SxaComponent } from '../sxa.component';
+import { Field, LinkField, SxaLinkListFields } from '@sitecore-jss/sitecore-jss-angular';
+
+@Component({
+  selector: 'app-link-list',
+  templateUrl: './link-list.component.html',
+})
+export class LinkListComponent extends SxaComponent<SxaLinkListFields> implements OnInit {
```

Comment:
SxaComponent is not a generic. 
This would error out. Did you test this locally?

Reply:
making SXA generic was supposed to be part of PR, I forgot to move the change into template.
Let me add it